### ISSUE TITLE
Alt text is not descriptive [../storage files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7072]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/storage/flutter/upload.mdx:
```
  115  
  116: ![S3 Metadata](/images/s3_metadata.png)
  117  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/prev/build-a-backend/storage/index.mdx:
```
  53  
  54: ![Image](/images/s3_overview.jpg)
  55  
```
